eitprocessing/datahandling/mixins/equality.py

```diff
@@ -62,8 +59,8 @@ def _array_safe_eq(a: Any, b: Any) -> bool:  # noqa: ANN401, PLR0911
             return False
 
     @staticmethod
-    def _remove_naming_info(d: dict | UserDict) -> dict:
-        x = ["label", "name", "description"]
+    def _allowed_inequality(d: dict | UserDict) -> dict:
```

Comment:
This method name is confusion to me. It returns a dict that represents data that are **not** allowed to be equal for an equality check to succeed. I think a better name would be either "_remove_allowed_inequalities()" (not great, because it does not remove anything) or "_equality_items()" or similar.

Reply:
I totally agree. I changed it now to `_return_equality_attributes`, which I still think is not an ideal name, but better than it was.
I would actually prefer handling this differently altogether, but that is definitely of scope of this PR. Dataclasses have a built-in system to recognize which attributes should and which should not be compared for equality checks. It would be great if we could use that option in our custom equality method, which would alleviate this method and make things much clearer. I have created #197 to deal with this